app/controllers/admin/basic_projects_controller.rb

```diff
@@ -24,7 +24,13 @@ def index
   def show
     @project = BasicProject.find(params[:id])
     authorize @project
-    prep_form_vars
+
+    case @tab = params[:tab] || "details"
+    when "details"
+      prep_form_vars
+    end
```

Comment:
This was my doing—there are going to be several more `when`s here as the other tabs get built later in this sprint.